compat/maven-settings-builder/src/test/java/org/apache/maven/settings/building/DefaultSettingsBuilderFactoryTest.java

```diff
@@ -32,17 +37,114 @@ private File getSettings(String name) {
         return new File("src/test/resources/settings/factory/" + name + ".xml").getAbsoluteFile();
     }
 
-    @Test
-    void testCompleteWiring() throws Exception {
+    private org.apache.maven.settings.Server asServer(Server delegate) {
```

Comment:
might be https://docs.pmd-code.org/pmd-doc-7.0.0-SNAPSHOT/pmd_rules_java_codestyle.html#unnecessaryfullyqualifiedname

```suggestion
    private Server asServer(Server delegate) {
```

Reply:
not used method ... 😄 I will drop

Reply:
yes thanks for this. Thats why we need proper tooling. Now we all see over and over.

- https://github.com/apache/maven/pull/2322